custom_components/wyzeapi/binary_sensor.py

```diff
@@ -44,19 +49,37 @@ async def async_setup_entry(
     client: Wyzeapy = hass.data[DOMAIN][config_entry.entry_id][CONF_CLIENT]
 
     sensor_service = await client.sensor_service
-    camera_service = await client.camera_service
+    #camera_service = await client.camera_service
+    irrigation_service = await client.irrigation_service
 
-    cameras = [
-        WyzeCameraMotion(camera_service, camera)
-        for camera in await camera_service.get_cameras()
-    ]
+    #cameras = [
+    #    WyzeCameraMotion(camera_service, camera)
+    #    for camera in await camera_service.get_cameras()
+    #]
     sensors = [
         WyzeSensor(sensor_service, sensor)
         for sensor in await sensor_service.get_sensors()
     ]
```

Comment:
I think if we are re-enabling the binary sensor platform we need to disable all the other binary sensors that are here and only add the irrigation one. I think the issue was over-polling with all these sensors so that's why the whole platform was disabled.

Reply:
I'd have to look into it more, but I'm not sure an update coordinator would work well in our case. It's more designed for a single endpoint to retrieve one big data dump and then send it to all the devices that use it. In our case we hit so many endpoints with specific device info in each call that it might take a crazy number of updaters.